src/modules/todos.js

```diff
@@ -0,0 +1,22 @@
+export default [
+  {
+    index: 1,
+    description: 'Going to university',
+    compeleted: false,
+  },
+  {
+    index: 2,
+    description: 'Doing math homework',
+    compeleted: false,
+  },
+  {
+    index: 3,
+    description: 'Visiting grandmom',
+    compeleted: false,
+  },
+  {
+    index: 4,
+    description: 'Hanging out with frinds',
+    compeleted: false,
+  },
+];
```

Comment:
I suggest you fix the method that iterates over this items since they do not show after the page loads. This is how your todo list currently displays:
<img width="1669" alt="Screenshot 2023-01-21 at 07 37 05" src="https://user-images.githubusercontent.com/48546992/213843670-d864bd7f-d798-40b0-ae6a-14227900d3ba.png">